### PR TITLE
[LOL-1644] Credit card icons added

### PR DESCRIPTION
**Pull request by ricalebro**
This PR adds credit card icons for the Wallet work:
https://www.figma.com/file/4PQp6K9dZPnkiUCW4GbpiN/Wallet%E2%80%94Web?node-id=336%3A2381

### CODE REVIEW

File: OTKit/otkit-icons/ic_card_american_express.svg [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<svg width="40" height="24" viewBox="0 0 40 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <title>Card/American Express</title>
```
Comment (ricalebro) [inline OTKit/otkit-icons/ic_card_american_express.svg:2]: Is there a specific format these title's should follow? I copied this format from the social buttons.

Comment (cosmonot1) [reply OTKit/otkit-icons/ic_card_american_express.svg:2]: @ricalebro No specific format. We use SVGO to clean up SVGs during the build phase

**Review by cosmonot1 — APPROVED** [review]
lgtm

**Comment by ronniechong** [comment]
The PR is failing because the icons are expected to be 24 x 24. These icons are 40 x 24.

**Comment by char1e5** [comment]
@ricalebro I could not find credit card icons in otkit icons library. Could you please confirm with designers to add this into otkit?

**Comment by ricalebro** [comment]
> @ricalebro I could not find credit card icons in otkit icons library. Could you please confirm with designers to add this into otkit? cc @rmorabia

I'm talking to design at the moment about where these should live, as they don't match the 24x24px design of the others, which is a requirement in the build right now.

**Comment by char1e5** [comment]
@ricalebro I got confirmation from design on this today. They don't want to have non-standard icons in otkit tokens. We can just add them in CFE to share across components.

**Comment by ricalebro** [comment]
Closing this as the icons will live in the cfe instead, see https://github.com/opentable/consumer-frontend/pull/4069

**Pull request closed without merging**